<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-12-02T16:01:24 EET" name="OpenCart Test Suite" finished-at="2021-12-02T16:01:53 EET" duration-ms="28768">
    <groups>
      <group name="regression">
        <method signature="TC_001_AccountRegistration.test_account_Registration()[pri:0, instance:testCases.TC_001_AccountRegistration@a38c7fe]" name="test_account_Registration" class="testCases.TC_001_AccountRegistration"/>
      </group> <!-- regression -->
      <group name="sanity">
        <method signature="TC_002_Login.test_Login()[pri:0, instance:testCases.TC_002_Login@7b36aa0c]" name="test_Login" class="testCases.TC_002_Login"/>
      </group> <!-- sanity -->
      <group name="master">
        <method signature="TC_001_AccountRegistration.test_account_Registration()[pri:0, instance:testCases.TC_001_AccountRegistration@a38c7fe]" name="test_account_Registration" class="testCases.TC_001_AccountRegistration"/>
        <method signature="TC_002_Login.test_Login()[pri:0, instance:testCases.TC_002_Login@7b36aa0c]" name="test_Login" class="testCases.TC_002_Login"/>
      </group> <!-- master -->
    </groups>
    <test started-at="2021-12-02T16:01:24 EET" name="OpenCart Tests" finished-at="2021-12-02T16:01:53 EET" duration-ms="28768">
      <class name="testCases.TC_002_Login">
        <test-method is-config="true" signature="setup(java.lang.String)[pri:0, instance:testCases.TC_002_Login@7b36aa0c]" started-at="2021-12-02T16:01:37 EET" name="setup" finished-at="2021-12-02T16:01:39 EET" duration-ms="1895" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="test_Login()[pri:0, instance:testCases.TC_002_Login@7b36aa0c]" started-at="2021-12-02T16:01:39 EET" name="test_Login" finished-at="2021-12-02T16:01:53 EET" duration-ms="14347" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at testCases.TC_002_Login.test_Login(TC_002_Login.java:53)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test_Login -->
      </class> <!-- testCases.TC_002_Login -->
      <class name="testCases.TC_001_AccountRegistration">
        <test-method is-config="true" signature="setup(java.lang.String)[pri:0, instance:testCases.TC_001_AccountRegistration@a38c7fe]" started-at="2021-12-02T16:01:25 EET" name="setup" finished-at="2021-12-02T16:01:30 EET" duration-ms="5272" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="test_account_Registration()[pri:0, instance:testCases.TC_001_AccountRegistration@a38c7fe]" started-at="2021-12-02T16:01:30 EET" name="test_account_Registration" finished-at="2021-12-02T16:01:37 EET" duration-ms="6733" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test_account_Registration -->
      </class> <!-- testCases.TC_001_AccountRegistration -->
    </test> <!-- OpenCart Tests -->
  </suite> <!-- OpenCart Test Suite -->
</testng-results>
